tests/conftest.py

```diff
@@ -12,6 +13,6 @@ def has_datarepo():
 
 @pytest.fixture(scope="session")
 def datarepo_dir():
-    """A namedtuple with the directory **absolute** paths for test data."""
+    """Named tuple with the directory **absolute** paths for test data."""
```

Comment:
the comment is wrong.(my bad)
is should just be Returns directory **absolute** paths for test data.
If you would like, please update it.